modules/auth/model/cms_temporal_user.e

```diff
@@ -0,0 +1,44 @@
+note
+	description: "Summary description for {CMS_TEMPORAL_USER}."
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	CMS_TEMPORAL_USER
+
+inherit
+
+	CMS_USER
+
+create
+	make,
+	make_with_id
+
+feature -- Access
+
+	personal_information: detachable STRING_32
```

Comment:
What about this "personal information" when the temp user becomes a normal user, do you store it in related user profile?